pkg/icingadb/ha.go

```diff
@@ -428,7 +429,7 @@ func (h *HA) realize(
 
 		h.signalTakeover(takeover)
 	} else if otherResponsible {
-		if state, _ := h.state.Load(); !state.otherResponsible {
+		if state := h.state.Load(); !state.otherResponsible {
```

Comment:
As we're now loading by-ref, `state.otherResponsible = true` below affects all readers. But this shouldn't be too bad as that's just a bool and we're storing it anyway right away.

Reply:
Fixed.